packages/components/src/components/MediaImage/MediaImage.stories.ts

```diff
@@ -0,0 +1,29 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Q: do we need to include this in this PR? As far as I know Altitude is planning on updating the regular image to use a Media type and having two `Image` components will be confusing
 for the users.
If it's just for testing we can use the custom component that is included in Colorful Coin:
<img width="246" alt="Screenshot 2024-02-28 at 13 15 47" src="https://github.com/contentful/experience-builder/assets/2773012/6cabce76-9e47-4394-8ce2-4308f862bcff">

Reply:
Thanks!  Noted!